BioArchLinux/meme/PKGBUILD

```diff
@@ -0,0 +1,74 @@
+#Maintainer: Kiri <kiri@vern.cc>
+#Contributor: allencch <allencch at hotmail dot com>
+#Contributor: Charles E. Vejnar
+pkgname=meme
+pkgver=5.5.3
+pkgrel=1
+pkgdesc="The MEME suite provides tools for discovering and using protein and DNA sequence motifs. https://doi.org/10.1093/nar/gkv416"
+arch=("i686"
+      "x86_64")
+url="http://meme-suite.org/"
+license=("custom")
```

Comment:
why one is custom and in tbtools is unknow?

Reply:
because this has license as resource !